docs/src/core/MERCHANT_INTEGRATION.md

```diff
@@ -132,7 +130,7 @@ For SPL Token transfers, use the `spl-token` parameter. The `spl-token` is the m
     const label = 'Jungle Cats store';
     const message = 'Jungle Cats store - your order - #001234';
     const memo = 'JC#4098';
-+   const splToken = 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v';
+    const splToken = 'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v';
```

Comment:
This needs to be a pubkey, see https://github.com/solana-labs/solana-pay/pull/49